testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/InternationalizationTest.java

```diff
@@ -53,29 +52,17 @@ public void addTestRealms(List<RealmRepresentation> testRealms) {
     @Before
     public void beforeI18nTest() {
         assertTestUserLocale(null);
-        assertEquals(DEFAULT_LOCALE_NAME, welcomeScreen.header().getCurrentLocaleName());
-    }
-
-    @Test
-    public void welcomeScreenTest() {
-        welcomeScreen.header().selectLocale(CUSTOM_LOCALE);
-        assertCustomLocaleWelcomeScreen();
-
-        // check if selected locale is preserved
-        welcomeScreen.clickPersonalInfoLink();
-        assertCustomLocaleLoginPage();
-        loginToAccount();
-        assertTestUserLocale(CUSTOM_LOCALE);
-        assertCustomLocalePersonalInfo();
     }
 
     @Test
     public void loggedInPageTest() {
         personalInfoPage.navigateTo();
         loginToAccount();
         assertTestUserLocale(null);
-        assertEquals(DEFAULT_LOCALE_NAME, personalInfoPage.header().getCurrentLocaleName());
-        personalInfoPage.header().selectLocale(CUSTOM_LOCALE);
+        personalInfoPage.selectLocale(CUSTOM_LOCALE);
+        personalInfoPage.clickSave(false);
+        WaitUtils.pause(1000);
```

Comment:
Please us this instead:
```suggestion
        WaitUtils.waitForPageToLoad();
```